library/init/meta/interactive.lean

```diff
@@ -661,50 +727,64 @@ tactic.trace_state
+This tactic applies to a goal such that its conclusion is an inductive type (say `I`). It tries to apply each constructor of `I` until it succeeds.
 -/
 meta def constructor : tactic unit :=
 tactic.constructor
 
-/-- Similar to `constructor`, but only non dependent premises are added as new goals. -/
+/-- 
+Similar to `constructor`, but only non-dependent premises are added as new goals. 
+-/
 meta def econstructor : tactic unit :=
 tactic.econstructor
 
+/--
+Applies the first constructor when the type of the target is an inductive data type with two constructors.
+-/
 meta def left : tactic unit :=
 tactic.left
 
+/--
+Applies the second constructor when the type of the target is an inductive data type with two constructors.
+-/
 meta def right : tactic unit :=
 tactic.right
 
+/--
+Applies the constructor when the type of the target is an inductive data type with one constructor.
+-/
 meta def split : tactic unit :=
 tactic.split
 
+/--
+Replaces the target of the main goal by `false`.
+-/
 meta def exfalso : tactic unit :=
 tactic.exfalso
 
 /--
-The injection tactic is based on the fact that constructors of inductive datatypes are injections.
-That means that if `c` is a constructor of an inductive datatype,
-and if `(c t₁)` and `(c t₂)` are two terms that are equal then  `t₁` and `t₂` are equal too.
+The `injection` tactic is based on the fact that constructors of inductive data types are injections. That means that if `c` is a constructor of an inductive datatype, and if `(c t₁)` and `(c t₂)` are two terms that are equal then  `t₁` and `t₂` are equal too.
 
-If `q` is a proof of a statement of conclusion `t₁ = t₂`,
-then injection applies injectivity to derive the equality of all arguments of `t₁` and `t₂` placed in the same positions.
-For example, from `(a::b) = (c::d)` we derive `a=c` and `b=d`.
-To use this tactic `t₁` and `t₂` should be constructor applications of the same constructor.
+If `q` is a proof of a statement of conclusion `t₁ = t₂`, then injection applies injectivity to derive the equality of all arguments of `t₁` and `t₂` placed in the same positions. For example, from `(a::b) = (c::d)` we derive `a=c` and `b=d`. To use this tactic `t₁` and `t₂` should be constructor applications of the same constructor.
 
-Given `h : a::b = c::d`, the tactic `injection h` adds to new hypothesis with types `a = c` and `b = d`
-to the main goal. The tactic `injection h with h₁ h₂` uses the names `h₁` an `h₂` to name the new
-hypotheses.
+Given `h : a::b = c::d`, the tactic `injection h` adds to new hypothesis with types `a = c` and `b = d` to the main goal. The tactic `injection h with h₁ h₂` uses the names `h₁` an `h₂` to name the new hypotheses.
 -/
 meta def injection (q : parse texpr) (hs : parse with_ident_list) : tactic unit :=
 do e ← i_to_expr q, tactic.injection_with e hs, try assumption
 
+/--
+`injections h₁ ... hₙ` iteratively applies `injection` to hypotheses using the names `h₁ ... hₙ`.
```

Comment:
Should be `injection with h₁ ... hₙ`, or perhaps the parameter should rather be changed to `parse ident*`? /cc @digama0

Reply:
Unless someone instructs me otherwise, I am going with `injections with` since that seems to be a closer parallel to the `injection` version.